OpenRA.Game/Network/OrderManager.cs

```diff
@@ -83,6 +83,13 @@ public void StartGame()
 			if (GameStarted)
 				return;
 
+			if (LobbyInfo.Clients.Count == 0)
```

Comment:
What edge case is this supposed to cover? I thought it might have been the shellmap, but that still works fine after commenting it out. Can we please add a comment explaining why this is needed, or remove it if not?